spectool/jdspec.ts

```diff
@@ -391,6 +392,10 @@ export function parseServiceSpecificationMarkdownToJSON(
                 case "report":
                 case "command":
                 case "const":
+                    // test for const foo = 0x12
+                    if (words.length === 4 && words[2] === "=") constant(words)
```

Comment:
this will parse "report x = 12" the same as "const x = 12"

also, i'm not sure I like the same keyword being used for two different things...

Reply:
now ``define x 12``